# a vector of joint angles
float64[6] joint_angles